verification/curator-service/ui/package.json

```diff
@@ -41,5 +42,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
Nit, but for now it's probably worth just using `dependencies`. There's not a meaningful distinction if our intention is to produce/serve a static bundle. See related discussion on CRA:

https://github.com/facebook/create-react-app/issues/4342

Reply:
fair enough, thanks for pointing this out. Is this a react only thing? For non ui packages we've used dev deps already, should we move everything over?

Reply:
It's not a React thing -- the distinction between devDeps and deps is simply not meaningful if the intention of your app is to compile into a static bundle rather than serve requests. It does make a difference if you're running a full-blown Node server.

For instance, we don't want to include @types in our production serving binaries, because it's useless bloat. So we use devDeps to indicate they aren't included in prod env. But our React code will never "serve" in prod, it'll just get compiled to a static bundle.

Reply:
ack, thanks for the clarification.